### PR TITLE
Add PeriodPojo, new test for DocumentResponse

### PR DESCRIPTION
**Pull request by kdoul**
Add PeriodPojo as a new ConceptValue, add new test in toop-regrep for the latest DocumentRequest.

### CODE REVIEW

File: toop-edm/src/main/java/eu/toop/edm/model/ConceptPojo.java [hunk 1]
```diff
@@ -287,6 +288,24 @@ public Builder valueMeasure (@Nullable final BigDecimal aValue, @Nullable final
       return value (ConceptValuePojo.builder ().measure (aValue, sUnitCode));
     }
 
+    @Nonnull
+    public Builder valuePeriod (@Nullable final PeriodPojo.Builder a)
```
Comment (phax) [inline toop-edm/src/main/java/eu/toop/edm/model/ConceptPojo.java:292]: Period is not part of the data model to my knowledge

File: toop-regrep/src/test/resources/examples/Document Response.xml [hunk 2]
```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (phax) [inline toop-regrep/src/test/resources/examples/Document Response.xml:1]: This file can only be read in toop-edm because additional XSDs are needed

Comment (kdoul) [reply toop-regrep/src/test/resources/examples/Document Response.xml:1]: Is there an example somewhere that reads a Document Response in the toop-edm?

Comment (phax) [reply toop-regrep/src/test/resources/examples/Document Response.xml:1]: Try the same setup as used for writing in https://github.com/TOOP4EU/toop-connector-ng/blob/master/toop-edm/src/test/java/eu/toop/edm/EDMResponseCreatorTest.java#L176

**Comment by phax** [comment]
@kdoul  can you make this PR please, but without the DocumentResponse example pls. Thx

**Comment by kdoul** [comment]
Yep! I will close this and push a fix asap

**Pull request closed without merging**